### ISSUE TITLE
Accessibility: <li> elements must be contained in a <ul> or <ol>

### ISSUE THREAD

**Issue by TomBonnerAtTFL** [Azure/api-management-developer-portal#1029]
Hi everyone, Tom here from Transport for London.

# Bug description

AXE doesn't consider the reports page accessible.

## Reproduction steps

1. Go to "Reports" page
2. Run ["AXE" accessibility testing tool](https://addons.mozilla.org/en-GB/firefox/addon/axe-devtools/).
3. AXE complains that `<li> elements must be contained in a <ul> or <ol>`

I'll be honest, this one I don't understand as much. Their advice is that the list item does not have a `<ul>, <ol> parent element without a role, or a role="list"`. So I think it doesn't like the fact that the parent "ul" has a role of "Navigation", which I suppose should be moved into a new wrapped DIV?

![image](https://user-images.githubusercontent.com/38533968/98566992-58d26b80-22a7-11eb-989a-322b3fd8014d.png)

## Expected behavior

All `<li>` elements should be contained within role-less `<ul>` element.

## Is your portal managed or self-hosted?

Managed

## API Management service name

N/A

## Environment

 - Operating system: Windows
 - Browser: Firefox
 - Version: Evergreen

## Additional context

Add any other context about the problem here, including screenshots.
